the-collab-lab/tcl-14-smart-shopping-list
# Yc je issue eight

This PR add principally a new functionality, the user can see through a checkbox if he/she has purchased an item in the last 24 hours

## Description

At first we had some problems with the methodology for handling the dates in React. But, we were able to solve it by bringing the dates from Firebase and applying a method to be able to compare it with another date and achieve the checkbox deactivation event after 24 hours.


## Related Issue



## Acceptance Criteria




## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |


## Updates

### Before


![Selection_012](https://user-images.githubusercontent.com/66391387/97767789-7e63b600-1aec-11eb-82ff-c47cf541bcd3.png)



### After


![Selection_013](https://user-images.githubusercontent.com/66391387/97767790-7f94e300-1aec-11eb-8211-66cef3ab473e.png)

## Testing Steps / QA Criteria
If you wanna testing this new functionality, first clone the repo and install node modules, after that you need to create a new token , later add an item to your list and that is all. You can see the box in an active state as long as 24 hours pass
